src/downloads.rs

```diff
@@ -87,8 +93,8 @@ pub fn download_and_extract(
     download(shell, &info, to, None)?;
 
     if let Some(name) = name {
-        shell.run(cmd!("mkdir -p {}", name).cwd(to))?;
-        shell.run(cmd!("tar -C {} --strip-components=1 -xvf {}", name, info.name).cwd(to))?;
+        shell.run(cmd!("mkdir -p {name}").cwd(to))?;
+        shell.run(cmd!("tar -C {name} --strip-components=1 -xvf {}", info.name).cwd(to))?;
```

Comment:
This looks fine to me, but why move `name` into the brackets, but not `info.name`?

Reply:
Currently, rust doesn't support putting expressions inline -- only identifiers.